core/modules/css/KvInterfaceImplMem.cc

```diff
@@ -92,14 +94,26 @@ KvInterfaceImplMem::KvInterfaceImplMem(string const& filename) {
 KvInterfaceImplMem::~KvInterfaceImplMem() {
 }
 
-void
-KvInterfaceImplMem::create(string const& key, string const& value) {
-    LOGF_DEBUG("create(%1%, %2%)" % key % value);
-    if (exists(key)) {
-        throw KeyExistsError(key);
+std::string
+KvInterfaceImplMem::create(string const& key, string const& value, bool unique) {
+    LOGF_DEBUG("create(%1%, %2%, unique=%3%)" % key % value % int(unique));
+
+    string path = key;
+    if (unique) {
+        // append unique suffix, in-memory KVI is not meant for large-scale
+        // storage, so we can do dumb brute-force loop
+        int seq = 0;
```

Comment:
I think the point here (and below?) is that random numbers are less likely to collide than sequential numbers, so they're faster to find a unique value.  (If the collision window is very short, then it doesn't matter, of course.)  There's no impact on testing or anything else since you can't know what unique value you would get anyway.